lib/SqlFormatter.php

```diff
@@ -214,7 +214,7 @@ protected static function getNextToken($string, $previous = null)
         }
 
         // Comment
-        if ($string[0] === '#' || (isset($string[1])&&($string[0]==='-'&&$string[1]==='-') || ($string[0]==='/'&&$string[1]==='*'))) {
+        if ($string[0] === '#' || (isset($string[1])&&($string[0]==='-'&&$string[1]==='-') || ($string[0]==='/'&&isset($string[1])&&$string[1]==='*'))) {
```

Comment:
whitespace?

Reply:
chtěl jsem držet aktuální coding style. kdybychom tu knihovnu adoptovali, tak bych to doladil, ale do tý doby se snažím zdržet coding style změn.

Reply:
no tak mas tam `$string[0] === '#'` a hned vedle `$string[0]==='-'` tak nevim co to je za coding style :) krome random

Reply:
jj, ale to jsem neměnil, přidával jsem jen `isset($string[1])&&` do poslední části podmínky